doc/whats_new/v0.21.rst

```diff
@@ -88,7 +88,11 @@ Support for Python 3.4 and below has been officially dropped.
 - |API| The ``n_components_`` attribute in :class:`cluster.AgglomerativeClustering`
   and :class:`cluster.FeatureAgglomeration` has been renamed to
   ``n_connected_components_``.
+
```

Comment:
seems like there's a newline at the wrong place

Reply:
yep, merged master too quickly...